src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -293,6 +293,10 @@ private ConfigProperties() {
     // How long (in seconds) to wait for job threads to finish during a graceful Tomcat shutdown
     public static final String ASYNC_JOBS_THREAD_SHUTDOWN_TIMEOUT = "candlepin.async.thread.shutdown.timeout";
 
+    // The maximum allowable size for pages; defaults to 3000
+    public static final String PAGING_MAX_PAGE_SIZE = "candlepin.paging.max_page_size";
```

Comment:
There's already a config property like this, the suggestion was to update its default value

Reply:
Oh good, I thought it was kinda odd I didn't see them when I searched for them. I must have typo'd "paging" while looking for the existing ones.

Fixed.